sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EQueryITest.scala

```diff
@@ -544,6 +544,8 @@ class SparkE2EQueryITest
       container.createItem(objectNode).block()
     }
 
+    Thread.sleep(1000)
```

Comment:
I fixed this already - will result in conflict (i used 2000 instead of 1000) - so you can probably revert this one